README.md

```diff
@@ -72,9 +61,7 @@ For troubleshooting information and installation directions for the entire Circu
 
 ### `SIMPLIFIED_CONFIGURATION_FILE`
 
-*Required in v1.1 only. Optional in v2.x.* The full path to configuration file in the container. Using the volume `-v` for v1.1, it should look something like `/etc/circulation/YOUR_CONFIGURATION_FILENAME.json`. In v2.x you can volume it in wherever you'd like.
-
-Use [this documentation](https://github.com/NYPL-Simplified/Simplified/wiki/Configuration) to create the JSON file for your particular library's configuration. If you're unfamiliar with JSON, you can use [this JSON Formatter & Validator](https://jsonformatter.curiousconcept.com/#) to validate your configuration file.
+*Optional.* The full path to configuration file in the container. Use [this documentation](https://github.com/NYPL-Simplified/Simplified/wiki/Configuration) to create the JSON file for your particular library's configuration. If you're unfamiliar with JSON, you can use [this JSON Formatter & Validator](https://jsonformatter.curiousconcept.com/#) to validate your configuration file.
```

Comment:
We're at the point where almost nobody needs to use the JSON configuration file, so marking it optional is good but I'd go so far as to say "you probably don't need this."